m/modem-manager.yml

```diff
@@ -0,0 +1,22 @@
+modem-manager:
+  image: ${REGISTRY_DOMAIN}/rancher/os-modemmanager:v1.5.0-rc1${SUFFIX}
```

Comment:
It's better to use this version format: `v${mm-version}-${patch-num}`, such as v1.1.2-1